charts/rosa-capi/values.yaml

```diff
@@ -1,5 +1,11 @@
 # Default values for rosa-capi.
 
+# Set the value to true if you need ArgoCd to be installed on the new cluster
+enable_argocd: false
+
+# Set namespace where you want the cluster to be installed
+namespace: default
```

Comment:
Using `default` may cause some challenges starting with the latest versions of OpenShift (especially for managed OpenShift) + not really a good practice to add content to the `default` namespace anyway. 

Having said that - the "namespace" approach may be something we do in a follow-up issue/PR as it may need a bit more structure around how it's managed anyway. If so, perhaps just leave it at `default` for now, but open a new issue in the backlog to work on the namespace management for CAPI.

Reply:
Added issue https://github.com/redhat-cop/declarative-openshift/issues/62